src/templates/base.html

```diff
@@ -223,6 +242,22 @@ <h4 class="ui inverted header">Footer Header</h4>
         }
         // Django returns <input type="hidden" value="csrf-token-secret"> so we just grab the `value` attribute from this
         CSRF_TOKEN = $('{% csrf_token %}').attr('value')
+
+        $(document).ready( function () {
+            $('.ui.search').search({
```

Comment:
make this jquery selector more specific (use an `#id` probably?) so we don't conflict with anything in `.tag` fils.

Reply:
✔️